packages/frontend/src/components/chart/ChartLoader.tsx

```diff
@@ -4,7 +4,7 @@ export function ChartLoader() {
   return (
     <svg
       data-role="chart-loader"
-      className="pointer-events-none relative z-20 mx-auto mt-[62px] hidden select-none opacity-40 sm:mt-[85px]"
+      className="pointer-events-none relative z-40 mx-auto mt-[62px] hidden select-none opacity-40 sm:mt-[85px]"
```

Comment:
comment would be useful, because I see some magic in there ;p

Reply:
I'm going to leave the reasoning here because it's straight forward I think and putting it as a comment is not really helpful.

`z-20` meant that the Loader element was behind the chart. `z-40` means that the Loader element is now in front of the Chart. Since DetailedTVL is opaque, we need all the things to be in front of the chart.